src/main/scala/BUILD

```diff
@@ -67,17 +68,15 @@ configure_zinc_scala(
 
 # Scala 3
 
+# Adding this, so we make sure to have a Scala library in the
+# IntelliJ libraries, so we can get a Scala SDK on sync.
 # Adding this, so we make sure to have a Scala library in the
 # IntelliJ libraries, so we can get a Scala SDK on sync.
 scala_library(
     name = "scala-sdk",
```

Comment:
Didn't have a Scala 3 SDK before because I forgot to add it.

The IntelliJ Scala + Bazel integration could use some improvements. Happy to chat more about this offline because I've got a fork of it that would work well with your toolchain work.